lib/platform/github/index.js

```diff
@@ -210,6 +210,17 @@ async function initRepo({
       );
       throw new Error('archived');
     }
+    const renovateConfig = JSON.parse(
```

Comment:
Uou still need the try/catch here. If it doesn't exist or we can't parse it, we want to catch and ignore the error